btc-address-generation/src/main/resources/address_generation_local.properties

```diff
@@ -7,6 +7,7 @@ btc-address-generation.notaryListSetterAccount=notary@notary
 btc-address-generation.changeAddressesStorageAccount=btc_change_addresses@notary
 btc-address-generation.threshold=2
 btc-address-generation.expansionTriggerAccount=expansion_trigger@notary
+btc-address-generation.expansionTriggerCreatorAccountId=superuser@bootstrap
```

Comment:
Do we really need to store it inside configs? If so, the same config must be added inside deposit and withdrawal services.

Reply:
Yes, otherwise anyone can send expansion transaction and services process it.